openvasd/openvasd.c

```diff
@@ -1591,6 +1589,56 @@ openvasd_get_health_started (openvasd_connector_t conn)
   return response;
 }
 
+openvasd_resp_t
+openvasd_get_performance (openvasd_connector_t conn,
+                          openvasd_get_performance_opts_t opts)
+{
+  openvasd_resp_t response = NULL;
+  gchar *err = NULL;
+  CURL *hnd = NULL;
+  struct curl_slist *customheader = NULL;
+  time_t now;
+  time (&now);
+
+  response = g_malloc0 (sizeof (struct openvasd_response));
+
+  if (!opts.titles || !strcmp (opts.titles, "") || opts.start < 0
+      || opts.start > now || opts.end < 0 || opts.end > now)
+    {
+      response->code = RESP_CODE_ERR;
+      response->body =
+        g_strdup ("{\"error\": \"Couldn't send get_performance command "
+                  "to scanner. Bad or missing parameters.\"}");
+      return response;
+    }
+  gchar *query =
```

Comment:
I'd like to see more consistency with the declarations. All the others are at the top of the block, even for example `hnd` which is first used below this.

Personally I'd also like you to drop the excess inits in general. Like response is set to NULL then it's set again basically straight away.